### ISSUE TITLE
[Bug Report] Lyrics desync

### ISSUE THREAD

**Issue by CompeyDev** [BobbyESP/Jetispot#24]
### Describe the bug

In certain cases, the lyrics view desyncs from the actual media content.

### To Reproduce

1. Play a song.
2. Either let it run in background for a while before checking the lyrics, or jump to a lyric and the lyrics view will not update to the media.

### Device info

Jetispot Version: 0.1.7
Android Version: 14
ROM: OxygenOS (OnePlus)

**Comment by CompeyDev** [comment]
Lmk if there are any logs or screen recordings I may need to provide.

**Comment by BobbyESP** [comment]
Jetispot codebase is really messy and really hard to maintain. It really has a lot work to put on and I don't have the enough time that I would like to fix it, so I don't know if I will be fixing this.

The media player implementation is bad, but understandable due to how hard is to reverse engineer Spotify 😆, and the lyrics sync works on top of this.

So sorry.

**Comment by CompeyDev** [comment]
In that case, I think it would be wise to put a maintainence notice, looking for maintainers who are willing to undertake large refactors.

**Comment by Outlet7493** [comment]
Agreed. However a rewrite might be easier than large refactors... :frowning: 

Happy to participate, but time is a constraint.
